File: 人机交互2020/9-4/9-4/1-1 VB.net入门/Test/frmTest.vb
Imports System.IO

Public Class frmTest

    Private Sub btnSeach_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnSeach.Click
        'lstInfo.Items.Clear()
        'lstInfo.Items.Add("aaaaa1")
        'lstInfo.Items.Add("aaaaa2")
        'lstInfo.Items.Add("aaaaa3")
        Dim path As String = "D:\0000人机交互"
        Dim files() As String = Directory.GetFiles(path)
        For i = 0 To files.Count - 1
            lstInfo.Items.Add(files(i))
        Next
        Dim dirs() As String = Directory.GetDirectories(path)
        For i = 0 To dirs.Count - 1
            lstInfo.Items.Add(dirs(i))
        Next
    End Sub
End Class

File: 人机交互2020/9-4/9-4/2-1 C#入门/Test/frmTest.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using System.IO;

namespace Test
{
    public partial class frmTest : Form
    {
        public frmTest()
        {
            InitializeComponent();
        }

        private void btnSearch_Click(object sender, EventArgs e)
        {
            //lstInfo.Items.Clear();
            //lstInfo.Items.Add("aaaa1");
            //lstInfo.Items.Add("aaaa2");
            //lstInfo.Items.Add("aaaa3");
            String path = "D:\\0000人机交互";
            String[] Dirs = Directory.GetDirectories(path);
            String[] Files = Directory.GetFiles(path);
            lstInfo.Items.Clear();
            for (int i = 0; i < Dirs.Count(); i++)
                lstInfo.Items.Add(Dirs[i]);
            for (int i = 0; i < Files.Count(); i++)
                lstInfo.Items.Add(Files[i]);
        }
    }
}
